### PR TITLE
User Context.

### PR DESCRIPTION
**Pull request by TheBluesBand**
First Time React Native Context.

How does it look tom?

### CODE REVIEW

File: contextManager.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import { createContext, useState } from 'react';
+import { Card } from './mock_data';
+
+//Interface Type for my 
+export interface user {
```
Comment (kebab01) [inline contextManager.tsx:5]: I would change this to upper case to make it clear that it is a type when we are importing it into other parts of the application

File: contextManager.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    setPhone: (phone: number) => void
+    setAddress: (address: string) => void
+    setPassword: (password: string) => void
+    setIsMember: (isMember: boolean) => void
+    addCard: (newCard: Card) => void
+}
+
+const UserContext = createContext<user | null>(null);
+
+function UserProvider({ children }: any) {
```
Comment (kebab01) [inline contextManager.tsx:26]: Separate the context provider and the user methods like setName, setEmail etc. I would create a hook called useUser(). Context manager stuff can go in this file and user stuff can go in the hook. You can refer to the Secure Assist user context or onboarding context.

**Review by kebab01 — COMMENTED** [review]
Muy bueno, algunas sugerencias.

**Review by kebab01 — APPROVED** [review]

**Review by kebab01 — APPROVED** [review]
muy bueno

**Comment by tynanmatthews** [comment]
Not merging as not needing with current back end implementation

**Pull request closed without merging**